packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -265,6 +267,26 @@ export default defineComponent({
     ...mapActions(['showMessage']),
     ...mapMutations('Files', ['SET_CURRENT_FOLDER', 'SET_FILE_SELECTION']),
 
+    showSpaceContextDrop(event, space) {
+      event.preventDefault()
+      const drop = this.$refs[`spaceContextDrop-${space.getDomSelector()}`]?.[0]
+
+      if (!drop) {
+        return
+      }
+
+      drop.tippy.setProps({
+        getReferenceClientRect: () => ({
+          width: 0,
+          height: 0,
+          top: event.clientY,
+          bottom: event.clientY,
+          left: event.clientX,
```

Comment:
Like for the `ResourceTable` the `left` and `right` prop needs to be set to bounding client rect `x` value of the context menu trigger button. Otherwise, using `Enter` with the keyboard opens the context menu in weird places.